src/devices/Ssd1331/Ssd1331/Ssd1331.cs

```diff
@@ -0,0 +1,1152 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Device.Gpio;
+using System.Device.Spi;
+using System.Drawing;
+using System.ComponentModel;
+using System.Runtime.InteropServices;
+
+namespace Iot.Device.Ssd1331
+{
+    public partial class Ssd1331 : IDisposable
+    {
+        #region Private Variables
+
+        private readonly int _dcPinId;
+        private readonly int _resetPinId;
+        private readonly bool _shouldDispose;
+
+        private SpiDevice _spiDevice;
+        private GpioController _gpioDevice;
+
+        private Color BGround_Color = Color.Black;
+        private Color Char_Color = Color.White;
+        int char_x;

```

Comment:
we're using explicit visibility so please add private.
And also the naming convention say we're stating those variables with underscore `_`.
Also, we don't use underscore to separate words but rather Pascal notations. So `char_x` would be `_charX` etc